path_follower:
  ros__parameters:
    lookahead: 1.0
    linear_velocity: 0.1
    angular_velocity: 0.3
    rotate_in_place_start_angle: 1.4
    rotate_in_place_end_angle: 0.3
    rotate_in_place_wait_time: 0.5
    driving_mode: hybrid
